packages/istanbul-api/lib/run-cover.js

```diff
@@ -23,7 +23,7 @@ function getCoverFunctions(config, includes, callback) {
         reportingDir = path.resolve(config.reporting.dir()),
         reporter = new Reporter(config),
         excludes = config.instrumentation.excludes(true),
-        coverageVar = '$$cov_' + new Date().getTime() + '$$',
```

Comment:
I'm wondering if the reason why coverage dropped in CI might be that this variable changed? what's the motivation for this edit.

Reply:
@bcoe The reason for this change here is that:
1.  I add some unit tests for hookRunInContext under istanbul-api (because there are some similar unit tests here) and
2. hookRunInContext treats ```___coverage__```  (rather than ```$$cov__*```) as the coverage variable because ```___coverage__``` is the actual coverage variable used by istanbul, and
3. istanbul-api/run-cover.js is called in those unit test, so that ```$$cov__*``` becomes the coverage variable in those tests.
4. In order to make the unit tests pass, I changed the coverageVar here.